Modules/System/Azure Function/README.md

```diff
@@ -0,0 +1,80 @@
+This module provides functionality for connecting to [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/).
+
+> This module does not store credentials for Azure Functions.  
+> Use caution when you store and pass credentials. 
+
```

Comment:
**NOTE:**
Azure Function Integration telemetry gathers data about the success or failure of calls from Business Central to Azure Functions. For more information, see [Analyzing Azure Function Integration Telemetry](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/telemetry-azure-function-integration-trace?branch=kepontop-azure-function-telemetry).